### PR TITLE
docs: Added Link/Reasons to join Discord Server.

### PR DESCRIPTION
**Pull request by MalwarePerseus**
docs: I've added the link as well as some reasons :wink:  to join the Discord Server! in the Need Help Section.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -381,7 +381,13 @@ file. To list all available commands, type `npm run`.
 [electron/electron-i18n]: https://github.com/electron/electron-i18n
 
 ## Need Help?
-
+### [Join](https://discord.gg/H6uTh7m) The Discord Server!
+Join the Discord server to get connected to thousands of Devs creating awesome projects with Electron.js everyday!
+  - Get Help! regarding any issue you are facing.
+  - Make new friends, working in the same space!
+  - Find some amazing projects to contribute to! or Help others with their issues!
+  - Still Reading for reasons to Join? duh! Just [Join](https://discord.gg/H6uTh7m)  Already!
```
Comment (VerteDinde) [inline CONTRIBUTING.md:389]:
```suggestion
Join the Discord server to get connected to thousands of devs creating awesome projects with Electron
  - Get help regarding any issue you are facing.
  - Make new friends, working in the same space
  - Find amazing projects to contribute to, or help others with their issues
```

**Review by VerteDinde — COMMENTED** [review]
Hey @MalwarePerseus, thanks so much for contributing to Electron and for your PR!

I made a few small changes to match our style guide (mostly removing exclamation points and letters that were capitalized). If these changes look okay to you, go ahead and accept them and we'll get your PR merged in. If you have any other changes you'd like to make or don't agree with what I wanted to change, let me know and we can talk here in the PR.

Thanks again for contributing!

File: CONTRIBUTING.md (hunk 1 above)
Comment (MalwarePerseus) [reply CONTRIBUTING.md:389]: Yes, sure, as it is much more compatible, with the rest of the file, it's great. 
Thanks for the changes.

**Review by VerteDinde — APPROVED** [review]
Looks great! Thanks again for contributing! 👍

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -381,7 +381,12 @@ file. To list all available commands, type `npm run`.
 [electron/electron-i18n]: https://github.com/electron/electron-i18n
 
 ## Need Help?
-
+### [Join](https://discord.gg/H6uTh7m) The Discord Server!
+Join the Discord server to get connected to thousands of devs creating awesome projects with Electron
```
Comment (erickzhao) [inline CONTRIBUTING.md:385]: hehe ambitious

**Review by erickzhao — APPROVED** [review]

**Pull request merged**